frontend/src/Navigation.ts

```diff
@@ -97,6 +97,13 @@ const calculateNavigation = async ({ dynamicNav, currentNamespace, base }: Dynam
     : routes;
 };
 
+/**
+ * Hook to be used by chrome to calculate navigation chunks.
+ * @param { dynamicNav, currentNamespace }
+ *  * dynamicNav - entry idicating dynamic navigation
+ *  * currentNamespace - current app namespace
+ * @returns either navigation object for nested items or array of navigation object for multiple entries.
+ */
 export const useNavigation = ({ dynamicNav, currentNamespace }: DynamicNav): Navigation | RouteProps[] => {
```

Comment:
This is interesting... This is the one the chrome is pulling from our module, right? Why is `calculateNavigation` the default export then? Not sure when this changed, but just noticed it now 😛

Reply:
It's for backwards compatibility, since we are already using it in chrome and chrome expects default export to be the previous async function I kept it there so it wouldn't break we merge and chrome won't